### PR TITLE
Load talents from client and bring back randomness

### PR DESCRIPTION
**Pull request by RubenSousaDinis**
## Summary

This PR makes some changes to the talent page:

- Random talents each day is back
- Load more add new rows to the end
- All filtering is now being done by the backend
- Add admin all view (this makes the All default view similar to all users)

### CODE REVIEW

File: app/packs/src/components/talent/TalentPage.jsx [hunk 1]
```diff
@@ -93,38 +106,61 @@ const TalentPage = ({ talents, pagination, isAdminOrModerator, env }) => {
...
-    return desiredTalent;
-  }, [localTalents, watchlistOnly, selectedSort, sortDirection]);
+    setTalents(sortedTalent);
+  }, [selectedSort, sortDirection]);
+
+  const loadTalents = (params) => {
+    setLoading(true);
+    get(`/api/v1/talent?${params.toString()}`).then((response) => {
+      if (response.error) {
+        toast.error(<ToastBody heading="Error!" body={response.error} />);
```
Comment (verdefred) [inline app/packs/src/components/talent/TalentPage.jsx:121]: should we also add this to a catch, if the call fails?

File: app/services/talents/search.rb [hunk 2]
```diff
@@ -45,10 +60,18 @@ def filter_by_keyword(talents)
       talents.where(user: users.distinct.select(:id))
     end
 
+    def filter_by_watchlist(talents)
```
Comment (verdefred) [inline app/services/talents/search.rb:63]: can we add a small test that touches on this method?

**Review by verdefred — COMMENTED** [review]
great work just left a couple of really small comments

File: app/services/talents/search.rb (hunk 2 above)
Comment (RubenSousaDinis) [reply app/services/talents/search.rb:63]: Added!

File: app/packs/src/components/talent/TalentPage.jsx (hunk 1 above)
Comment (RubenSousaDinis) [reply app/packs/src/components/talent/TalentPage.jsx:121]: we're not doing that in other parts of the codebase. I prefer to keep it similar

**Review by verdefred — APPROVED** [review]

**Pull request merged**